### ISSUE TITLE
Create AsyncTasks for all network tasks that happen in the UI thread for honeycomb compatibility

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#223]
```
Honeycomb throws a 
[http://developer.android.com/reference/android/os/NetworkOnMainThreadException.
html NetworkOnMainThreadException] if there happens to be network activity 
invoked from the main/ui thread. We should use 
[http://developer.android.com/reference/android/os/AsyncTask.html AsyncTask]'s 
to avoid this.

One area where this needs to be fixed is 
[http://code.google.com/p/gtalksms/source/browse/src/com/googlecode/gtalksms/pan
els/About.java#35 here], there are may be others too.

```

Original issue reported on code.google.com by `fschm...@gmail.com` on 27 Sep 2011 at 5:20

**Comment by GoogleCodeExporter** [comment]
```
About panel uses now AsynTask for URL fetching:
http://code.google.com/p/gtalksms/source/detail?r=f76d026320d0f53682a535355a090e
72ae06484f
It would be nice if we had a loading spinner in the upper right corner of the 
about panel.

The DNS SRV lookups in the wizard still need to be put in an AsyncTask.
```

Original comment by `fschm...@gmail.com` on 5 Oct 2011 at 10:31
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
All SRV lookups are now within a thread: 
http://code.google.com/p/gtalksms/source/detail?r=134329b253df2054eeff7c161c1c8e
17307b349a

Leaving eye-candy for later
```

Original comment by `fschm...@gmail.com` on 18 Oct 2011 at 8:04
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed and published with 3.3
```

Original comment by `Florent....@gmail.com` on 11 Nov 2011 at 9:47
- Changed state: **Deployed**
